cmd/upgrade/app/v1alpha1/executor.go

```diff
@@ -40,26 +49,71 @@ type ExecutorBuilder struct {
 //It adds object in ExecutorBuilder struct with the help of config
 func ExecutorBuilderForConfig(cfg *apis.UpgradeConfig) (b *ExecutorBuilder) {
 	b = &ExecutorBuilder{}
+
+	instanceName := os.Getenv(instanceNameENVKey)
+	if instanceName == "" {
+		b.errors = append(b.errors,
+			errors.Errorf("failed to instantiate executor builder: %s ENV not present", instanceNameENVKey))
```

Comment:
Can we indent to make it more readable?
Can we rephrase as below:
```go
"failed to instantiate executor builder: ENV {%s} is not set", envSelfName
```

Reply:
updated.